AFQ/segmentation.py

```diff
@@ -563,10 +570,13 @@ def segment_afq(self, tg=None):
         fgarray = np.array(_resample_tg(tg, 100))
         n_streamlines = fgarray.shape[0]
 
-        streamlines_in_bundles = np.zeros(
+        streamlines_in_bundles = np.empty(
```

Comment:
Should this be a boolean array set to all-False on initialization instead?

Reply:
This name I think is confusing and maybe should be changed. It really stores the probability of being in that bundle or 0/np.nan if not in the bundle. So it can have multiple bundle probabilities per streamline. It is only collapsed down to a boolean array on line 700: `possible_fibers = np.nansum(streamlines_in_bundles, -1) > 0`